tock/employees/models.py

```diff
@@ -96,34 +96,39 @@ class UserData(models.Model):
     is_18f_employee = models.BooleanField(default=True, verbose_name='Is 18F Employee')
     is_billable = models.BooleanField(default=True, verbose_name="Is 18F Billable Employee")
     unit = models.IntegerField(null=True, choices=UNIT_CHOICES, verbose_name="Select 18F unit", blank=True)
+    float_people_id = models.CharField(max_length=50, null=True, blank=True, verbose_name='Float "people_id" attribute')
```

Comment:
I _believe_ the decision was based on what was coming across from Float as the `people_id` via the [/people](https://github.com/floatschedule/api/blob/master/Sections/people.md) end point. Maybe I was future proofing in case there was a change in the content of the people_id to include non-integer characters? As you noted below, I tried to control for a blank people_id (that is, Tock pinged Float to find info a user but returned `''` when it couldn't find a match) in the `save()` function.

Of course happy to flip it to an `IntegerField` if you think it is better practice!